source/data/tier1data/clients/mango_portal.rst

```diff
@@ -0,0 +1,179 @@
+
+If you made a mistake, you can click on 'Remove file' under the file in question. 
+When you are ready, click on 'Start uploading files' to upload your selection. 
+
+To download a data object, from the page of that object you click on the download-icon next to its name.
+Alternatively, you can download it from the page of its parent collection by clicking on the download
+icon next at the end of its row.
+
+.. figure:: ../images/mango_portal/mango_portal_download.png
+  :width: 800
+  :alt: Downloading files via the ManGO portal
+
+Uploads and downloads via the ManGO portal are limited to 1GB and 20GB per file respectively.
+You can upload multiple files at the same time, but you can only download data objects individually.
+It is currently not possible to upload or download entire folders at once.
+If you want to transfer larger amounts of data via a graphical interface, you can use `Globus <https://vlaams-supercomputing-centrum-vscdocumentation.readthedocs-hosted.com/en/latest/globus/globus_main_index.html>`_.
+
+.. _edit-permissions:
+
+***********
+Permissions
+***********
+
+To view the :ref:`permissions<collaboration>` on a collection or data object, click on it and then go to the tab 'Permissions'. 
+
+.. figure:: ../images/mango_portal/mango_portal_permissions.png
+  :width: 800
+  :alt: An overview of the permissions on a collection
+
+
+If you have 'own' permissions yourself, you can add new permissions at the bottom of the page, remove permissions by clicking on the trash bin icon,
+and switch on/off the inheritance for collection permissions.
+
+You can give permissions to any group that you are member of. 
+To do so, select the group and the rights you want to give from their respective dropdown menus and click on 'apply.'
+If you are applying permissions to a collection, you can also indicate whether to apply the permissions recursively.
+
+.. _edit-metadata:
+
+**************
+Metadata
+**************
+
+Every collection or data object has its own :ref:`metadata<metadata>` tab.
+When you click on this tab, you can see all metadata which is added to the object. 
+
+.. figure:: ../images/mango_portal/mango_portal_metadata_overview.png
+  :alt: getting an overview of the metadata on an object
+
+
+If only manual metadata is added, you get one overview of all the metadata.
+However, if the metadata comes from multiple sources, the overview is split into tabs:
+
+- Metadata added via schemas can be found in the tab with the name of the respective schema     
+- Metadata added via automatic extraction can be found in the tab 'analysis'  
+- Manually added metadata can be found in the tab 'other'   
+
+
+On the right ride of each AVU, you may see the icons of a blue pencil and a red trashbin.
+Clicking on the former allows you to overwrite the AVU, while the latter allows you to delete it.
+If you do not have rights to edit/delete this metadata, these buttons may be absent.   
+
+
+Adding metadata manually
```

Comment:
I would suggest to add something mentioning that although this is possible and useful for some use cases it is not the recommended way to add metadata as it is error prone and does not enforce standarization

Reply:
This is IMO not something for this page, but something for the metadata page. I'll add it in the next iteration, so it's also on the mango documentation